thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/HolidayEventsPipeline.java

```diff
@@ -55,15 +60,36 @@ public HolidayEventsPipeline(String outputName, Set<String> inputNames, Map<Stri
   @Override
   public PipelineResult run(PipelineContext context) {
     TimeRangeEntity current = TimeRangeEntity.getContextCurrent(context);
-
-    EventFilter filter = new EventFilter();
-    filter.setEventType(EventType.HOLIDAY.toString());
-    filter.setStartTime(current.getStart());
-    filter.setEndTime(current.getEnd());
+    long currentStart = new DateTime(current.getStart()).minusDays(2).getMillis();
+    long currentEnd = current.getEnd();
+    long baselineStart = new DateTime(currentStart).minusDays(BASELINE_OFFSET).getMillis();
+    long baselineEnd = new DateTime(currentEnd).minusDays(7).getMillis();
+    TimeRangeEntity baseline = TimeRangeEntity.fromRange(1.0, TimeRangeEntity.TYPE_BASELINE, baselineStart, baselineEnd);
```

Comment:
Change is intentional, shabby numbers are not. Fixed this to use constants. Added to javadoc to explain why this is done